acceptance/api/v1/application_deploy_test.go

```diff
@@ -171,7 +171,7 @@ var _ = Describe("AppDeploy Endpoint", func() {
 				deployResponse := deployApplication(appName, namespace, deployRequest)
 
 				// check also the random suffix for the default route: '-[a-z0-9]{5}'
```

Comment:
remove the comment too?

Reply:
I should, yes

Reply:
Done 1dbc3a45c0dd90bb67c50e92a15e88837cc76ba5